generic/sosreport.py

```diff
@@ -42,9 +42,7 @@ def setUp(self):
         dist = distro.detect()
         sm = SoftwareManager()
         sos_pkg = ""
-        if not process.system("sosreport", ignore_status=True, sudo=True):
-            self.log.info("sosreport is installed")
-        elif dist.name == 'Ubuntu':
+        if dist.name == 'Ubuntu':
```

Comment:
@harish-24 Would you want to handle `supportconfig` in SuSE ?

Reply:
Would that be included in sos test? @PraveenPenguin Your thoughts please.

Reply:
@harish-24 i think supportconfig having different set of options so i think it should be cover on different test case. it help us to maintain test case in well.

Reply:
@PraveenPenguin @harish-24 Sure then. Thanks